README.md

```diff
@@ -1,13 +1,17 @@
-# Toit Flutter Demo
+# Flutter Demo
```

Comment:
Suggest:
Demo of controlling IoT devices with Flutter (using Toit.io).

Reply:
done. without the "(using Toit.io)".